plugins/restful/RestfulEntityBase.php

```diff
@@ -324,15 +355,17 @@ public function viewEntity($id) {
         }
 
         if (empty($info['formatter'])) {
-          if ($sub_wrapper instanceof EntityListWrapper) {
-            // Multiple values.
-            foreach ($sub_wrapper as $item_wrapper) {
-              $value[] = $this->getValueFromProperty($wrapper, $item_wrapper, $info, $public_field_name);
+          if (field_is_translatable($this->entityType, field_info_field($info['property']))) {
+            $enabled_languages = array_keys(language_list());
+
+            foreach ($enabled_languages as $language) {
+              $wrapper->language($language);
+              $sub_wrapper = $info['wrapper_method_on_entity'] ? $wrapper : $wrapper->{$property};
```

Comment:
(Out of scope) TODO: Create a method to get the sub-wrapper.

I'm going to create a follow up issue.